src/utils/object/omit.test.ts

```diff
@@ -0,0 +1,16 @@
+import { omit } from "utils/object";
+
+describe("omit", () => {
+  it("returns an object without the supplied key(s)", () => {
+    expect(omit({} as { a?: number }, ["a"])).toMatchObject({});
```

Comment:
I think we could omit this test. This type assertion is misleading since it doesn't affect the real type of the object.